### PR TITLE
New int: ComputerPower + small fixes

### PR DESCRIPTION
**Pull request by dot-Justin**
`integrations/computer.py`
  - New int! ComputerPower will lock|sleep|shutdown|restart your machine via LAMatHome.
  - Experimental Mac support, as I don't have a Mac to test this on.
  - Also cleaned up unused code.

`utils/llm_parse.py`
  - General cleanup
  - Allow the prompt access variables like `googlehome_automations`, as we somehow weren't using f strings before 👍
  - Add support for ComputerPower integration
  - Reordered function in `CombinedParse` to be in the order of the file list, alphabetically
 
`integrations/google.py`
  - Cleaned up code
  - Optimized GoogleHome function by clicking a "don't show again" box in a dialogue on first login, which speeds it up in the long run.

`config.json`
  - Reformatted to make it easier to read (I hope)

`main.py`
  - General bugfix (check if task != "x" for correct logging)

`config.json`
  - Reformatted to make it easier to read (I hope)

```
 (\   /)

(◠ + ◠ )
   ►◄
```

### CODE REVIEW

File: integrations/google.py [hunk 1]
```diff
@@ -33,14 +38,38 @@ def GoogleHome(page, automation):
             page.click("text=Not now")
             logging.info("Clicked 'Not now' for simplify sign-in prompt")
 
-        page.click("text=OK")
-        logging.info("Clicked 'OK' after logging in")
-        page.wait_for_load_state('load')
+    if first_login:
+        try:
+            time.sleep(2)
+            # page.wait_for_selector("label:has-text(\"Don't show again\")", timeout=300)
```
Comment (pnivek) [inline integrations/google.py:44]: You meant to uncomment this right? if not can you remove?

Comment (dot-Justin) [reply integrations/google.py:44]: Meant to remove. good catch!

Comment (dot-Justin) [reply integrations/google.py:44]: Removed this and the one on line 52. Pushing soon

Comment (pnivek) [inline integrations/google.py]: sir...

**Review by pnivek — APPROVED** [review]
lgtm sir

File: integrations/google.py (hunk 1 above)
Comment (dot-Justin) [reply integrations/google.py]: Hit a ctrl+r for me SIR 😆

**Pull request merged**